src/classlibnative/inc/nlsinfo.h

```diff
@@ -101,10 +101,8 @@ class COMNlsInfo {
     //
     static FCDECL1(FC_BOOL_RET, nativeInitCultureData, CultureDataBaseObject *data);
     static FCDECL3(FC_BOOL_RET, nativeGetNumberFormatInfoValues, StringObject* localeNameUNSAFE, NumberFormatInfo* nfi, CLR_BOOL useUserOverride);
-#ifdef FEATURE_USE_LCID
```

Comment:
yes and no. for this one I just removed it because it looks the defines not get passed to the compiler. I thought about force passing it but while I am working on enabling the code under this ifdef I found it is really missed up. for example the code get enabled is referencing some stuff under #ifndef coreclr which make the mix and match very hard. what I am trying to do here is to get the code enabled and then at some point we need to clean up the whole FEATUTRE_USE_LCID

---
In reply to: [75936022](https://github.com/dotnet/coreclr/pull/6866#discussion_r75936022) [](ancestors = 75936022)